scopesim/tests/tests_integrations/test_flux_is_conserved_through_full_system.py

```diff
@@ -82,3 +82,70 @@ def test_flux_is_conserved_for_yes_bg_emission(self, non_unity_cmds, tbl_src):
         assert src_flux == approx(1)          # u.Unit("ph s-1")
         assert np.sum(im - np.median(im)) == approx(0.45, rel=1e-2)
         assert np.median(im) == approx(1.5, abs=1e-2)
+
+
+@pytest.mark.usefixtures("protect_currsys", "patch_all_mock_paths")
+class TestStackedStars:
+    """Test whether stars can be stacked."""
+
+    def test_stacked_stars(self):
+        """Test whether stars can be stacked.
+
+        Four stacked faint stars should have the same magnitude as one bright star.
+        """
+        stars = _table_source_overlapping()
+        temp_celsius = 5.0
+        dit = 10  # s
+        ndit = 1
+        s_filter = "H"
+        cmd = UserCommands(use_instrument="basic_instrument")
+
+        cmd.update(properties={
+            "!OBS.ndit": ndit,
+            "!OBS.dit": dit,
+            "!OBS.airmass": 1.0,
+            "!ATMO.background.filter_name": s_filter,
+            "!ATMO.temperature": temp_celsius,
+            "!TEL.temperature": temp_celsius,
+            "!DET.width": 512,  # pixel
+            "!DET.height": 512,
+        })
+        micado = OpticalTrain(cmd)
```

Comment:
I was gonna say why use MICADO here and not try to use the `basic_instrument`, but then I realized you are, it's just the variable name ^^

Reply:
Yeah, I noticed the bad naming too after making the PR. I would have changed the variable name if I noticed it earlier, but thought it wasn't important enough.

Reply:
Nah, it's not important especially in the tests...